core/src/main/java/org/apache/carbondata/core/datastore/columnar/BlockIndexerStorageForNoDictionary.java

```diff
@@ -18,18 +18,14 @@
 package org.apache.carbondata.core.datastore.columnar;
 
 import java.util.Arrays;
-import java.util.Map;
 
 import org.apache.carbondata.core.metadata.datatype.DataType;
 
+/**
+ * Only try applying RLE on rowId
+ */
 public class BlockIndexerStorageForNoDictionary extends BlockIndexerStorage<Object[]> {
```

Comment:
Please change class name to a more understandable one

Reply:
done